core/crypto/client_tcert_pool_mt.go

```diff
@@ -63,7 +63,26 @@ func (tCertPool *tCertPoolMultithreadingImpl) Stop() (err error) {
 	return
 }
 
-func (tCertPool *tCertPoolMultithreadingImpl) GetNextTCert() (tCert tCert, err error) {
+func (tCertPool *tCertPoolMultithreadingImpl) GetNextTCerts(nCerts int) (tCerts []tCert, err error) {
+	// The MT is considered WIP. We are going to add this support in the next round of refactoring, but added this
```

Comment:
I assume this comment implies that the implementation of this method is incomplete? And you just expect the execution of this code to hit the first error case for nCerts != 1? Otherwise, I don't understand how you are returning an error in the case on ln 75 and then again on ln 81?